mmap-alloc/src/lib.rs

```diff
@@ -339,7 +341,11 @@ impl MapAlloc {
             }
 
             *new_ptr = first_byte;
-            ptr::copy_nonoverlapping(ptr::null().offset(1), new_ptr.offset(1), allocator.pagesize - 1);
+            ptr::copy_nonoverlapping(
+                ptr::null().offset(1),
```

Comment:
If dereferencing a null pointer is UB on LLVM, might this be as well? Might be safer to just do `1 as *mut _`. I'm not sure though.